src/standard.yml

```diff
@@ -120,6 +120,9 @@ jobs:
     parameters:
       executor:
         type: executor
+      store_screenshots:
+        type: boolean
+        default: false
```

Comment:
why not make this default to true?

Reply:
I think that would require projects with `report_coverage: true` to manually set `store_screenshots` to false.  Otherwise they'd be setting 2 different paths for artifacts, and I'm not sure if that's a problem or not.

Reply:
Looks like multiple `store_artifacts` steps is [not a problem](https://circleci.com/docs/2.0/artifacts/#uploading-artifacts).